platform/android/java/editor/src/main/java/org/godotengine/editor/GodotEditor.kt

```diff
@@ -272,6 +273,11 @@ open class GodotEditor : GodotActivity() {
 	 */
 	protected open fun enablePanAndScaleGestures() =
 		java.lang.Boolean.parseBoolean(GodotLib.getEditorSetting("interface/touchscreen/enable_pan_and_scale_gestures"))
+	/**
+	 * Define which axis of mouse wheel event will be used for mapping RotaryInput.
```

Comment:
The javadoc should be updated to match the added documentation.

Reply:
Fixed.